Makefile

```diff
@@ -351,7 +351,7 @@ CXXFLAGS_FOR_BUILD :=
 LDFLAGS_FOR_BUILD  :=
 
 else ifeq ($(UNAME),Darwin)
-ifeq ($(shell sw_vers -productName),macOS)
+ifeq ($(shell sw_vers -productName),macOS) # Swap to Mac OS X for devices older than Big Sur
```

Comment:
I'd also like to note that I should find a different way to detect macOS vs iOS if this is an issue.